tests/test_evalscript.py

```diff
@@ -0,0 +1,137 @@
+from typing import List, Optional
+
+import pytest
+
+from sentinelhub import CRS, BBox, DataCollection, MimeType, SentinelHubRequest, generate_evalscript
+
+
+@pytest.mark.parametrize("data_collection", [DataCollection.SENTINEL2_L1C, DataCollection.SENTINEL1_IW])
+def test_collection_bands(data_collection: DataCollection) -> None:
+    evalscript = generate_evalscript(data_collection=data_collection)
+    expected_bands_str = "bands: [" + ", ".join(f'"{b.name}"' for b in data_collection.bands) + "]"
+    assert expected_bands_str in evalscript
+
+
+@pytest.mark.parametrize(
+    "data_collection, bands",
+    [
+        (DataCollection.SENTINEL2_L2A, ["B04", "B03", "B02"]),
+        (DataCollection.SENTINEL1_IW, ["VV", "VH"]),
+    ],
+    ids=str,
```

Comment:
wont this make really long names? perhaps the name can be the data collection?

Reply:
It's not so long, and I prefer to know which test I'm running in vscode.

![image](https://user-images.githubusercontent.com/37869250/229447006-97c789bf-ed3a-494c-8e6e-4ebd30ef72d0.png)

If you still think it's too ugly/long, I can revert